JavaScript spacing tweaks

cliffordp opened the pull request:
Based on my understanding, looking like the following is correct:
```
(function( $ ) {
...
})( jQuery );
```

cliffordp committed:
JavaScript spacing tweaks

Based on my understanding, looking like the following is correct:
```
(function( $ ) {
...
})( jQuery );
```

Viper007Bond commented:
Based on https://make.wordpress.org/core/handbook/best-practices/coding-standards/javascript/, it sounds like this change is warranted. I'm out on medical leave though so I don't have time to test and verify.

cliffordp committed:
additional JS tweaks per docs

Additional mods per https://make.wordpress.org/core/handbook/best-practices/coding-standards/javascript/

Yes, `FORCE_QUOTE_STYlE` is correct capitalization. No, `FORCE_QUOTE_STYLE` does not also work. #BlamePhpStorm

cliffordp commented:
Thanks for that link. For some reason, at the time of adding this, I wasn't sure where it was.

I added more JS rules based on that reading. PhpStorm doesn't have options to make it match exactly; for example, WP wants a space after `!` but not before `++` and therefore I left the "Unary operators" box unchecked, which means a space after `!` will not be added.

I'd encourage you, as the expert, to double-check all these and make sure all the other coding standards are also correct in PhpStorm and then generate your own export file to compare (feel free to start with my pull request, as I think it's correct).

Thanks, and I hope all medical things are good-to-go soon! :)

cliffordp commented:
@Viper007Bond ping
